packages/analytics/src/integrations/Omnitracking/definitions.ts

```diff
@@ -664,7 +664,10 @@ export const trackEventsMapper: Readonly<OmnitrackingTrackEventsMapper> = {
     filtersApplied: getFilterParametersFromEvent(data.properties),
   }),
   [EventType.InteractContent]: (data: EventData<TrackTypesValues>) => {
-    if (data.properties?.interactionType === InteractionType.Scroll) {
+    if (
+      data.properties?.interactionType === InteractionType.Scroll &&
+      data.properties.contentType !== 'carousel'
```

Comment:
As far as I understood the PR, `contentType` is to be replace with `elementType`. If that is the case, shouldn't this be based on the `elementType` property instead of the `contentType` one? Or if you want to keep backwards compatibility, accept both?

Reply:
Updated this _if_ to check both `contentType` and `elementType`. Fixed 👍🏼